weave-js/src/components/PagePanelComponents/Home/Home.tsx

```diff
@@ -59,13 +72,6 @@ const HomeComp: FC<HomeProps> = props => {
   const isAuthenticated = useIsAuthenticated();
   const userEntities = query.useUserEntities(isAuthenticated);
   const userName = query.useUserName(isAuthenticated);
-  const [activeBrowserRoot, setActiveBrowserRoot] = useState<
-    | undefined
-    | {
-        browserType: 'recent' | 'wandb' | 'drafts';
-        rootId: string;
-      }
-  >(undefined);
   const recentSection = useMemo(() => {
     if (RECENTS_SUPPORTED) {
       return [
```

Comment:
seems like for all these the `onClick` is not needed anymore?

Reply:
I think removing onClick should be possible but will require some larger changes to rewrite how setPreviewNode works

Reply:
👍